### ISSUE TITLE
Home Page Stats counter broken.

### ISSUE THREAD

**Issue by prkhandelwal** [dotnet/dotnetfoundation-website#43]
The stats counter values keep decreasing each time you scroll us and scroll down again.

**Comment by jongalloway** [comment]
Can't reproduce - I think this might have been an issue during the deployment. Are you still seeing this?

**Comment by prkhandelwal** [comment]
Yes the issue is still present. I am attaching a link to video reproducing the bug.

https://1drv.ms/v/s!AmPSCdCSeks4j8kgIl0svnIgLTb9TA

**Comment by andrewcahill** [comment]
Yeah you will only see it once the browser window is smaller than the height between the top of the counters and the top of the browser window - effectively running this counter each time you scroll from top to bottom.

There is a javascript function within : https://github.com/dotnet/dotnetfoundation-website/blob/master/src/wwwroot/js/home.js between lines 5-25 to determine this range and will initiate the counter animation.

**Comment by prkhandelwal** [comment]
The issue is, each time I scroll down from the top the range of counter keeps decreasing.

**Comment by prkhandelwal** [comment]
, zq, . S.... 
###
